plasma_framework/contracts/src/framework/utils/ExitPriority.sol

```diff
@@ -3,21 +3,32 @@ pragma solidity ^0.5.0;
 import "../../utils/TxPosLib.sol";
 
 library ExitPriority {
+
     /**
-     * @dev formula of priority is as folllowed: (exitableAt || txPos || nonce).
-     * The first 64 bit for exitableAt, following 128 bits of txPos and then 64 bits of nonce.
-     * The combinition of 'exitableAt' and 'txPos' is the priority for Plasma M(ore)VP protocol.
-     * 'exitableAt' only provide granuality of block, thus add 'txPos' to provide priority of transaction.
+     * @dev Given a utxo position and a unique ID, returns an exit priority.
+     * The combination of 'exitableAt' and 'txPos' is the priority for Plasma M(ore)VP protocol.
+     * 'exitableAt' only provide granularity of block, thus add 'txPos' to provide priority of transaction.
+     * @param exitId Unique exit identifier.
+     * @return An exit priority.
+     *   Anatomy of returned value, most significant bits first
+     *   42 bits - timestamp (exitable_at); unix timestamp fits into 32 bits
+     *   54 bits - blknum * 10^9 + txindex; to represent all utxo for 10 years we need only 54 bits
```

Comment:
this is incorrect since it is using `txPos` instead of `utxoPos` so actually have 10*4 more spaces here compare to the original one in `RootChain.sol`.

Reply:
but we really want here only tx position - which output exits is determined in exit id

Reply:
> to represent all utxo for 10 years we need only 54 bits

ah...I mean the comment itself is incorrect, not the txPos

Reply:
heh, right!